Client/src/Components/TabPanels/Status/GeneralSettingsPanel.jsx

```diff
@@ -0,0 +1,281 @@
+			const buffer = 12;
+			setProgress((prev) => {
+				if (prev.value + buffer >= 100) {
+					clearInterval(intervalRef.current);
+					return { value: 100, isLoading: false };
+				}
+				return { ...prev, value: prev.value + buffer };
+			});
+		}, 120);
+	};
+
+	return (
+		<TabPanel
+			value="General Settings"
+			sx={{
+				"& h1, & p, & input": {
+					color: theme.palette.text.tertiary,
+				},
+			}}
+		>
+			<Stack
+				component="form"
+				className="status-general-settings-form"
+				noValidate
+				spellCheck="false"
+				gap={theme.spacing(12)}
+				mt={theme.spacing(6)}
+			>
+				<ConfigBox>
+					<Box>
+						<Stack gap={theme.spacing(6)}>
+							<Typography component="h2">Access</Typography>
+							<Typography component="p">
+								If your status page is ready, you can mark it as published.
+							</Typography>
+						</Stack>
+					</Box>
+					<Stack gap={theme.spacing(6)}>
+						<Checkbox
+							id="published-to-public"
+							label={`Published and visible to the public`}
+							isChecked={localData.publish}
+							value={"localData.publish"}
+							onChange={handleChange}
+							onBlur={handleBlur}
+						/>
+					</Stack>
+				</ConfigBox>
+
+				<ConfigBox>
+					<Box>
+						<Stack gap={theme.spacing(6)}>
+							<Typography component="h2">Basic Information</Typography>
+							<Typography component="p">
+								Define company name and the subdomain that your status page points to.
+							</Typography>
+						</Stack>
+					</Box>
+					<Stack gap={theme.spacing(18)}>
+						<TextInput
+							id="companyName"
+							type="text"
+							label="Company name"
+							value={localData.companyName}
+							onChange={handleChange}
+							onBlur={handleBlur}
+							helperText={errors["companyName"]}
+							error={errors["companyName"] ? true : false}
+						/>
+						<TextInput
+							id="url"
+							type="url"
+							label="SubURL"
+							value={localData.url}
+							startAdornment={<HttpAdornment https={true} />}
+							//prefix={"http://uptimegenie.com/"}
+							onChange={handleChange}
+							onBlur={handleBlur}
+							helperText={errors["url"]}
+							error={errors["url"] ? true : false}
+						/>
+					</Stack>
+				</ConfigBox>
+				<ConfigBox>
+					<Box>
+						<Stack gap={theme.spacing(6)}>
+							<Typography component="h2">Appearance</Typography>
+							<Typography component="p">
+								Define the default look and feel of your public status page.
+							</Typography>
+						</Stack>
+					</Box>
+					<Stack gap={theme.spacing(6)}>
+						<Select
+							id="display-timezone"
+							label="Display timezone"
+							value={localData.timezone}
+							onChange={handleChange}
+							onBlur={handleBlur}
+							items={timezones}
+							error={errors["display-timezone"]}
+						/>
+						<ImageField
+							id="logo"
+							src={localData.logo?.src ?? logo?.src}
+							loading={progress.isLoading && progress.value !== 100}
+							onChange={handleLogo}
+						/>
```

Comment:
This is the field used for the Avatar I believe, I'm not sure if we want this to be a circular image as with the avatar?  @gorkem-bwl can you confirm design spec here?

Reply:
Is this about the company logo? If so, it is a rectangular shape as can be seen in the Figma file, and not circular. It will be located on top left of the public page, FYI @jennifer-gan

Reply:
Yes, I missed that, I will change that to rectangle

Reply:
Fixed in : Fixed here: https://github.com/bluewave-labs/checkmate/pull/1307/commits/c301656a7aeb61a6739d4510f1587932dd81bfd4